splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/actions/IndexConstantOperation.java

```diff
@@ -204,28 +206,29 @@ public String getScopeName() {
         List<Integer> indexCols = tentativeIndex.getIndex().getIndexColsToMainColMapList();
         List<Integer> allFormatIds = tentativeIndex.getTable().getFormatIdsList();
         List<Integer> columnOrdering = tentativeIndex.getTable().getColumnOrderingList();
-        td.getColumnDescriptorList().elementAt(0).getType().getNull();
-        int[] rowDecodingMap = new int[allFormatIds.size()];
-        int[] baseColumnMap = new int[allFormatIds.size()];
+        List<Integer> storagePositions = tentativeIndex.getTable().getStoragePositionsList();
+        int maxStoragePosition = Collections.max(indexCols);
```

Comment:
It would be more intuitive to me if `maxStoragePosition = Collections.max(storagePositions)`. Not sure if `indexCols` is the same as `storagePositions` at this point.

Reply:
IndexCols stores storage positions